plugins/kuota-calc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuota-calc
+spec:
+  version: v0.1.0
+  homepage: https://github.com/postfinance/kuota-calc
+  shortDescription: Calculate the maximum needed resource quota for k8s objects.
```

Comment:
typically it's not great to include words like `k8s` here as it's obvious.
you could rewrite as:

> Calculate needed space to perform rolling updates

but you can fix it later.

Reply:
Will do, thank you!